src/components/modal/lagrede-filter/lagrede-filter-nytt.tsx

```diff
@@ -0,0 +1,76 @@
+import {Input} from "nav-frontend-skjema";
+import {Hovedknapp} from "nav-frontend-knapper";
+import React, {useEffect, useState} from "react";
+import {useDispatch, useSelector} from "react-redux";
+import {AppState} from "../../../reducer";
+import {Normaltekst} from "nav-frontend-typografi";
+import {erTomtObjekt, feilValidering} from "./lagrede-filter-utils";
+import {STATUS} from "../../../ducks/utils";
+import {LagretFilterValideringsError} from "./lagre-filter-modal";
+import {ErrorModalType, LagredeFilterVarselModal} from "./varsel-modal";
+import {lagreNyttFilter} from "../../../ducks/lagret-filter";
+
+export function LagreNytt(props: { lukkModal}) {
+    const filterValg = useSelector((state: AppState) => state.filtreringMinoversikt)
+    const {status, data} = useSelector((state: AppState) => state.lagretFilter)
+    const [filterNavn, setFilterNavn] = useState("")
+    const [saveRequestSent, setSaveRequestSent] = useState(false)
+    const [errorModalErApen, setErrorModalErApen] = useState(false)
+    const [feilmelding, setFeilmelding] = useState({} as LagretFilterValideringsError)
+    const dispatch = useDispatch();
+    const lukkModal = props.lukkModal
+
+
+    useEffect(() => {
+        if (saveRequestSent) {
+            if (status === STATUS.PENDING) {
+            } else if (status === STATUS.ERROR) {
+                setErrorModalErApen(true)
+                setSaveRequestSent(false)
+            } else if (status === STATUS.OK) {
+                setErrorModalErApen(false)
+                setSaveRequestSent(false)
+                lukkModal()
+            }
+        }
+    }, [lukkModal, saveRequestSent, status])
+
+    const doLagreNyttFilter = (event) => {
+        event.preventDefault()
+        const trimmetFilterNavn = filterNavn.trim()
+        const feilValideringResponse = feilValidering(trimmetFilterNavn, data)
```

Comment:
maybe feilValidering should do filterNavn.trim() ?

Reply:
yep

Reply:
Thanks, fixed.